cmd/backupmanager/backupmanager_test.go

```diff
@@ -38,16 +38,19 @@ import (
 var binaryPath string
 
 func init() {
-	var ok bool
+	wd, err := os.Getwd()
+	binaryPath = filepath.Join(wd, "..", "..", "bin")
 
-	binaryPath, ok = os.LookupEnv("BIN_DIR")
-	if !ok {
-		wd, err := os.Getwd()
-		binaryPath = filepath.Join(wd, "..", "..", "bin")
+	if err != nil {
+		panic(err.Error())
+	}
 
-		if err != nil {
-			panic(err.Error())
-		}
+	// Always rebuild backupmanager
+	bashCmd := "cd " + binaryPath + " && (rm backupmanager || true) && go build ../cmd/backupmanager"
```

Comment:
You could use `rm -f` to avoid ugly `|| true`

Reply:
Nope:
```
$ rm -r nosuchfile
rm: nosuchfile: No such file or directory
$ echo $?
1
```

Reply:
I believe you are right, tnx.

Reply:
What I personally dislike about -f is that it may remove a write-protected file.

Reply:
Yeah, but `backupmanager ` could be write-protected?
Your point is fine when you are trying to delete a user input data but in this case you we could ignore "write-protected" part.